crates/compiler/can/src/suffixed.rs

```diff
@@ -60,19 +70,17 @@ fn init_unwrapped_err<'a>(
             // Provide an intermediate answer expression and pattern when unwrapping a
             // (sub) expression.
             // e.g. `x = foo (bar!)` unwraps to `x = Task.await (bar) \#!a0 -> foo #!a0`
-            let answer_ident = arena.alloc(next_unique_suffixed_ident());
+            let ident = arena.alloc(format!("{}_arg", next_unique_suffixed_ident()));
```

Comment:
Why did we add the `_arg`?

Reply:
Just for clearer purpose of the variable. The value comes from the argument